types/wordpress__components/angle-picker-control/index.d.ts

```diff
@@ -0,0 +1,24 @@
+import { ComponentType } from 'react';
+
+declare namespace AnglePickerControl {
+    interface Props {
+        /**
+         * The current value of the input. The value represents an angle in degrees and should be a value between 0 and 360.
+         */
+        value: number;
+
+        /**
+         * Label to use for the angle picker. If not set the a translated label "Angle" is used.
+         */
+        label: string;
```

Comment:
[Docs say this isn't required](https://github.com/WordPress/gutenberg/blob/aa0abb915e843e2d636d4242aa249e504284abbc/packages/components/src/angle-picker-control/README.md#label), but I haven't confirmed (it's just passed to `BaseControl`).

```suggestion
        label?: string;
```